Cargo.toml

```diff
@@ -25,6 +25,7 @@ winapi = ["windows-link"]
 std = ["alloc"]
 clock = ["winapi", "iana-time-zone", "android-tzdata", "now"]
 now = ["std"]
+core_error = []
```

Comment:
Nit: replace the `_` with a `-` please? Otherwise this makes sense to me.

Reply:
done!